src/apps/service/apps/de/listings.clj

```diff
@@ -97,7 +97,7 @@
    :id         trash-category-id
    :name       "Trash"
    :is_public  true
-   :total      (count-deleted-and-orphaned-apps params)})
+   :total      (delay (count-deleted-and-orphaned-apps params))})
```

Comment:
This is a delay now, because otherwise it always executes and it takes ages, and can't get cancelled due to the transaction, it seems. Really, I think the trash category stuff only happens when the category is listed directly, so it won't affect much.